xExtension-ColorfulList/static/script.js

```diff
@@ -0,0 +1,40 @@
+$(document).ready(function(){
```

Comment:
This probably isn't necessary when you append a script to the end of the document so I'd remove it completely (after testing), but fyi that's jQuery for:

```suggestion
document.addEventListener('DOMContentLoaded', function(){
```